scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -932,7 +944,7 @@ Function RunLongHaulTest
 
     $testStartAt = Get-Date
     $deviceId = "${ReleaseLabel}-Windows-${Architecture}-longHaul"
-    (Get-Content $DeploymentWorkingFilePath).replace('<Analyzer.DeviceID>',$deviceId) | Set-Content $DeploymentWorkingFilePath
+    (Get-Content $DeploymentWorkingFilePath).replace('<DeviceID>',$deviceId) | Set-Content $DeploymentWorkingFilePath
```

Comment:
Need to roll back?

Reply:
Yes, we could also get this param from the environment in the analyzer.